.golangci.yml

```diff
@@ -20,7 +20,7 @@ linters:
   disable-all: true
   enable:
     - bodyclose
-    - deadcode
+    # deadcode
```

Comment:
Was `deadcode` also complaining about some parts? If so, we should rather fix the issue in the code.